### PR TITLE
Replace org.json by Jackson and Lombok

### PR DESCRIPTION
**Pull request by seofernando25**
This PR passes the task of the current JSON to Object conversion to the Jackson library.

Note that most of the code found in [the config directory](https://github.com/uorocketry/rocket-ground-station/blob/fasterxml_lombok_impl/src/main/java/uorocketry/basestation/config) was made by https://app.quicktype.io/ just pasted the `config.json` file and got most of the type definitions.

Unfortunately, most of the changes are just calls to the `getters` and formatting (eg: removing unused imports, long lines, etc)

**TLDR; This is a way too long PR, ignore**

### CODE REVIEW

**Comment by seofernando25** [comment]
stale
